examples/react-client/use-camera-and-microphone/src/AdditionalControls.tsx

```diff
@@ -40,29 +34,52 @@ export const AdditionalControls = () => {
       {show && (
         <div className="flex flex-row flex-wrap gap-2 p-2 md:grid md:grid-cols-2">
           <div className="grid grid-cols-2 gap-2">
-            <DeviceControls
-              device={camera}
-              type={"video"}
-              status={peerStatus}
-            />
+            <div className="flex flex-col gap-2">
+              <button
+                className="btn btn-success btn-sm"
+                onClick={() => {
+                  camera.toggleCamera();
+                }}
+              >
+                Toggle camera (it's now {camera.isCameraOn ? "on" : "off"})
```

Comment:
**question:** Is this correct?

Reply:
i'd say yes, this is to let the user know what will happen upon a click (earlier there was two buttons, for on and off)